lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -49,13 +50,17 @@ const useFetchGrade = student => {
       const fetchData = async () => {
         setGradeLoading(true);
         setGrade(''); // Clear previous grade so we don't show the wrong grade with the new student
-
-        const response = await fetchGrade({ student, authToken });
-        if (!didCancel) {
-          // Only set these values if we didn't cancel this request
-          if (response.currentScore) {
-            setGrade(scaleGrade(response.currentScore, GRADE_MULTIPLIER));
+        try {
+          const response = await fetchGrade({ student, authToken });
+          if (!didCancel) {
+            // Only set these values if we didn't cancel this request
+            if (response.currentScore) {
+              setGrade(scaleGrade(response.currentScore, GRADE_MULTIPLIER));
+            }
           }
+        } catch (e) {
+          setFetchGradeError(e.errorMessage ? e.errorMessage : 'Unknown error');
```

Comment:
```suggestion
          setFetchGradeError(e.errorMessage ?? 'Unknown error');
```

Reply:
I think that won't work because errorMessage could be just an empty string IIRC.